QUICKSTART.md

```diff
@@ -0,0 +1,308 @@
+# OpenTelemetry QuickStart
+
+<!-- Re-generate TOC with `markdown-toc --no-first-h1 -i` -->
+
+<!-- toc -->
+
+- [Configuration](#configuration)
+    + [Sampler](#sampler)
+    + [Span Processor](#span-processor)
+    + [Exporter](#exporter)
+- [Tracing](#tracing)
+  * [Create basic Span](#create-basic-span)
+  * [Create nested Spans](#create-nested-spans)
+  * [Span Attributes](#span-attributes)
+  * [Create Spans with events](#create-spans-with-events)
+  * [Create Spans with links](#create-spans-with-links)
+  * [Context Propagation](#context-propagation)
+- [Metrics](#metrics)
+
+<!-- tocstop -->
+
+OpenTelemetry can be used to instrument code for collecting distributed traces and recording metrics.
+For more details, check out the [Specification Overview].
+
+In the following examples, we demonstrate how to configure the OpenTelemetry SDK, create spans and record metrics through the OpenTelemetry API.
+
+[Specification Overview]: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/overview.md
+
+# Configuration
+
+**Libraries** that want to export distributed tracing using OpenTelemetry only need a dependency on the `opentelemetry-api` package
+and should never configure OpenTelemetry themselves. The configuration must be provided by **Applications** which should also depend on the 
+`opentelemetry-sdk` package, or any other implementation of the OpenTelemetry API. This way, libraries will obtain a real tracer
+implementation only if the user application is configured for it. For more details, check out the [Library Guidelines].
+
+[Library Guidelines]: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/library-guidelines.md
+
+The application has to install a span processor with an exporter and may customize the behavior of the OpenTelemetry SDK.
```

Comment:
May want to note that this configuration info only applies to the SDK provided by `opentelemetry-sdk` since you mention "any other implementation" above. E.g. there's no `updateActiveTraceConfig` in the API package, and other SDKs are free to offer different configuration options.